### DISCUSSION TITLE
Html

### DISCUSSION THREAD

**Discussion by BaraAlqady123** [community/community#33504] [Other Feature Feedback, Questions, & Ideas]
How to make html page

**Comment by ihnatenkoo** [comment]
Hello, see here 
https://www.w3schools.com/html/html_editors.asp

**Comment by iLakshya** [comment]
Hey @BaraAlqady123 

Okay so since you are totally new to this, then I would recommend that please understand the basics of HTML through this link:
https://www.w3schools.com/html/default.asp

Try cool stuff and make your website through HTML and CSS. Basically, we use CSS for styling and designing as it is a bad practice to write any code in HTML Tags. Hence, we create CSS for styling and link it to the GTML folder we are working on using <link> tag. To make the website responsive, we use media queries, say suppose you have your phone, your laptop and your tablet, then you might have noticed that certain websites have different layouts on each device, that is known as responsiveness. Though we can add styling in the <style> tag but still I prefer you to make a separate folder with .css as an extension and link with HTML.

Making the website responsive is the very last task. First, create a basic layout, then if you are satisfied with your layout, add responsiveness to it. If you want to add any other things then you can add them through. 

Now, if you want to create your website dynamic and interactive then learn JS. But, I don't recommend that right now as you are in your learning phase and JS is a bit tricky so leave that for now. Focus on HTML and CSS first, then move forward to JS.

These are the best websites that I have used personally and I would recommend you as well to learn from these as these are easy to understand:
Link for HTML: https://www.w3schools.com/html/default.asp
Link for CSS: https://www.w3schools.com/css/default.asp
Link for JS: https://www.w3schools.com/js/default.asp

I am also attaching the cheatsheets of HYML, CSS, and JS

[JavaScript Cheatsheet.pdf](https://github.com/community/community/files/9604053/JavaScript.Cheatsheet.pdf)

[HTML Cheatsheet.pdf](https://github.com/community/community/files/9604055/HTML.Cheatsheet.pdf)

[CSS Cheatsheet.pdf](https://github.com/community/community/files/9604057/CSS.Cheatsheet.pdf)


Happy Learning ✌🏻✨

**Comment by avin-madhu** [comment]
first of all understand the basics of HTML like:
- HTML tags
- attributes

you can start by referring:  [html tutorial](https://www.w3schools.com/html/)

download  any code editor for writing HTML code ( for eg: VS Code, Sublime Text or just Notepad)

happy Learning 😄
